### DISCUSSION TITLE
Erratic behavior with 11:CC9 and 11:CC16 triggering by themselves

### DISCUSSION THREAD

**Discussion by designdrop** [rsjaffe/MIDI2LR#554] [Problems]
Recently updated everything and now there is an issue where 11:CC9 and 11:CC16 are triggering by themselves. 
Using an XTouch Mini but pretty sure that is not what is causing it because I disconnected all peripherals and it is still happening
Everything is up to date Windows 11 LR11  M2LR 5.1.1.0

![2021-11-16_20-11-19_SparkVideo](https://user-images.githubusercontent.com/8203611/142134552-51862588-f714-442d-a0f9-ced2cebfcd33.gif)

**Comment by rsjaffe** [comment]
Include the contents of the log file (MIDI2LR.log) in your message. This file is found in %AppData%\MIDI2LR.

Also, what MIDI programs are you running? Have you tried replacing the USB cable connecting the XTouch to your computer?

**Reply by designdrop** [reply]
Well that wasn't too hard =)
It was an app called **LoopBe Internal MIDI**. It has a mute button and toggling that fixed it. 

Thank you!
